2002-11-27  Jakub Jelinek  <jakub@redhat.com>

	* config/tc-ppc.c (ppc_cleanup): Remove SEC_MERGE.

--- gas/config/tc-ppc.c.jj	Fri Sep 27 11:33:09 2002
+++ gas/config/tc-ppc.c	Wed Nov 27 11:37:13 2002
@@ -1329,7 +1329,7 @@ ppc_cleanup ()
     apuinfo_secp = subseg_new (".PPC.EMB.apuinfo", 0);
     bfd_set_section_flags (stdoutput,
 			   apuinfo_secp,
-			   SEC_HAS_CONTENTS | SEC_READONLY | SEC_MERGE);
+			   SEC_HAS_CONTENTS | SEC_READONLY);
 
     p = frag_more (4);
     md_number_to_chars (p, (valueT) 8, 4);
